api_app/script_analyzers/observable_analyzers/dnstwist.py

```diff
@@ -0,0 +1,49 @@
+import subprocess
+import json
+from shutil import which
+
+from api_app.exceptions import AnalyzerRunException
+from api_app.script_analyzers import classes
+
+
+class DNStwist(classes.ObservableAnalyzer):
+    command: str = "dnstwist"
+
+    def set_config(self, additional_config_params):
+        self._geoip = additional_config_params.get("geoip", False)
+        self._ssdeep = additional_config_params.get("ssdeep", False)
+        self._mxcheck = additional_config_params.get("mxcheck", False)
+        self._tld = additional_config_params.get("tld", False)
+        self._tld_dict = additional_config_params.get("tld_dict", "abused_tlds.dict")
+
+    def run(self):
+        if not which(self.command):
+            self.report["success"] = False
+            raise AnalyzerRunException("dnstwist not installed!")
+
+        args = [self.command, "--registered", "--format", "json"]
+        if self._geoip:
+            args.append("--geoip")
+        if self._ssdeep:
+            args.append("--ssdeep")
+        if self._mxcheck:
+            args.append("--mxcheck")
+        if self._tld:
+            args.append("--tld")
+            args.append(self._tld_dict)
+
+        args.append(self.observable_name)
+
+        process = subprocess.Popen(
+            args,
+            stdout=subprocess.PIPE,
+            stderr=subprocess.PIPE,
+        )
+        process.wait()
+        stdout, stderr = process.communicate()
+
+        dns_report = stdout.decode("utf-8"), stderr
+        dns_str = dns_report[0]
+        dns_json = json.loads(dns_str)
```

Comment:
You could check the value in `stderr` and trigger an `AnalyzerRunException` in the case is populated.

Also, maybe you could verify that the output is supported JSON. Otherwise it would crash unexpectedly